homeassistant/components/systemair/climate.py

```diff
@@ -0,0 +1,224 @@
+"""Support for the SystemAIR HVAC."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.climate import PLATFORM_SCHEMA, ClimateDevice
+from homeassistant.components.climate.const import (
+    ATTR_FAN_MODE, ATTR_HVAC_MODE, ATTR_PRESET_MODE, FAN_HIGH, FAN_LOW,
+    FAN_MEDIUM, FAN_OFF, HVAC_MODE_AUTO, HVAC_MODE_OFF, SUPPORT_FAN_MODE,
+    SUPPORT_PRESET_MODE, SUPPORT_TARGET_TEMPERATURE)
+from homeassistant.const import ATTR_TEMPERATURE, CONF_NAME, TEMP_CELSIUS
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from homeassistant.components.systemair import (
+    ATTR_CURRENT_HUMIDITY, ATTR_TARGET_TEMPERATURE,
+    PRESET_AUTO, PRESET_CROWDED, PRESET_FIREPLACE, PRESET_HOLIDAY, PRESET_IDLE,
+    PRESET_MANUAL, PRESET_REFRESH, SIGNAL_SYSTEMAIR_UPDATE_DONE)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
```

Comment:
Please remove this schema. We should load the platforms via discovery from the component.
https://github.com/home-assistant/home-assistant/blob/614cf7422596569464f5fe2699c49721b2ae4de8/homeassistant/helpers/discovery.py#L127